tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+import os
```

Comment:
added this __init__.py file to allow me to more easily run the test modules from the command line, the import statements were failing otherwise. adding a source_path setup for running tests from the parent directory.